Gemfile

```diff
@@ -10,3 +10,5 @@ end
 platforms :jruby do
   gem 'jruby-openssl'
 end
+
+gem "rexml", "~> 3.2"
```

Comment:
if we need a min version on this, it should go in the gemspec as a development dependency. I believe these others are here because there isn't a good way to do platform-specific deps in a gemspec. It's questionable if we even need those these days.